src/database/models/course-topic-content.ts

```diff
@@ -32,6 +32,9 @@ export default class CourseTopicContent extends Model implements CourseTopicCont
     public deadDate!: Date;
     public partialExtend!: boolean;
 
+    public lastExported!: Date | null;
```

Comment:
Why not embed this in the export url?

Reply:
I guess because the export url doesn't exist until the export is done

Reply:
We needed something to differentiate when the process to export has never started and when it is in progress.